backend/src/main/java/com/team4/backend/controller/FileMetaDataController.java

```diff
@@ -43,15 +40,15 @@ protected String getLoggedUserName(Principal loggedUser) {
 
     @GetMapping("/getListInvalidCvNotSeen/{noPage}")
     @PreAuthorize("hasAuthority('INTERNSHIP_MANAGER')")
-    public Flux<FileMetaDataInternshipManagerViewDto> getListInvalidCvNotSeen(@PathVariable Integer noPage) {
+    public Flux<FileMetaDataInternshipManagerViewDto> getListInvalidCvNotSeen(@PathVariable Integer noPage) throws InvalidPageRequestException {
         return fileMetaDataService.getListInvalidCvNotSeen(noPage).map(FileMetaDataMapper::toInternshipManagerViewDto);
     }
 
     @PatchMapping("/validateCv")
     @PreAuthorize("hasAuthority('INTERNSHIP_MANAGER')")
     public Mono<ResponseEntity<String>> validateCv(@RequestParam("id") String id, @RequestParam("isValid") Boolean isValid) {
         return fileMetaDataService.validateCv(id, isValid)
-                .flatMap(fileMetaData -> Mono.just(ResponseEntity.ok().body("")))
```

Comment:
Ok (200) serait mieux ici, car NO CONTENT est qu'une ressource est mise à jour, sans modifier le contenu actuel de la page affichée à l'utilisateur. Dans validateCV, on enlève un C.V. lorsque la requête est envoyé

Reply:
On enlève un C.V. de la vue*

Reply:
yeah you're right